smithy-swift-codegen-test/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+plugins {
+    id("software.amazon.smithy").version("0.5.3")
+}
+
+description = "Smithy protocol test suite"
```

Comment:
Is this new test suite related to plugins or something else entirely?

Reply:
I have removed the `smithy-swift-codegen-test` directory

Reply:
I think it would be nice to have a codegen specific test suite, to see what artifacts generic`smithy-swift` can generate, but out of scope for this PR.